lib/gen_stage.ex

```diff
@@ -718,8 +718,12 @@ defmodule GenStage do
   @typedoc "The supported stage types."
   @type type :: :producer | :consumer | :producer_consumer
 
-  @typedoc "The supported init options."
-  @type options :: keyword()
+ @typedoc "Option values used by the `init*` functions"
+  @type option :: {:max_demand, non_neg_integer} |
+                  {:min_demand, non_neg_integer}
+
+  @typedoc "Options used by the `init*` functions"
+  @type options :: [option]
```

Comment:
Should be `option()`.

Reply:
Why? I'm actually wondering that myself right now why parentheses are necessary. They seemed to be used irregularly and `mix test` seemed to allow it.

Reply:
Parens do not matter on typespecs.

Reply:
It's just a convention now I'm pretty sure... Elixir warns now when calling things like 

```elixir
defmodule Example do
  def test, do: some_method
  def some_method, do: true
end
```

```
warning: variable "some_method" does not exist and is being expanded to "some_method()", please use parentheses to remove the ambiguity or change the variable name
  iex:2
```

Reply:
NVM